cli/src/cmd/pkg/validate.go

```diff
@@ -0,0 +1,84 @@
+package pkg
+
+import (
+	"cli/core"
+	"cli/util"
+	"path"
+	"strings"
+
+	"github.com/luno/jettison/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	ErrNoConfigImage            = errors.New("config file missing field 'image'")
+	ErrNoPackages               = errors.New("no packages or custom packages specified")
+	ErrUndefinedProfilePackages = errors.New("packages in profile not in any of packages, custom-packages, or command-line custom-packages")
+)
+
+func validate(cmd *cobra.Command, config *core.Config) error {
```

Comment:
What you could also add is a validate function that validates the config against the json schema (yml schema) to rule out schema issues, rather than checking it manually, there should be a package for it in go too, I did it in the kafka-mapper.